ibu-imager/cmd/restore.go

```diff
@@ -47,8 +47,15 @@ func restore() {
 	hostCommandsExecutor := ops.NewNsenterExecutor(log, true)
 	op := ops.NewOps(log, hostCommandsExecutor)
 
+	successfulStages = map[string]bool{
```

Comment:
I don't think we should use this option, we should reuse same logic as we do in creator or remove run.once logic in creator and move to this one. Lets not use 2 different ways for setting if we run function or not

Reply:
the problem I see with using run.once is that right now there are a couple of steps (i.e. recert validations and stopping cluster services) that don't use it... these two are important to track for selective cleanup later on

Reply:
Can we validate that we should run recert or we should start containers? For example currently if there was an issue with stopping containers and etcd is still up cleanup will try to spawn etcd and stuck till timeout, so we could check for example if cluster is up and running before trying to cleanup it? 
I mean having to different options to set Done will be very confusing and will break in a week

Reply:
Agree on maintaining just one method. Waiting for #181 to land, so we can adjust this to use the `*.done` file approach.